build-ios-shared.sh

```
@@ -10,29 +10,21 @@ set -e
 cd $dir
-onnxruntime_version=${SHERPA_ONNX_ONNXRUNTIME_VERSION:-1.27.0}
+onnxruntime_version=${SHERPA_ONNX_ONNXRUNTIME_VERSION:-1.27.1}
 onnxruntime_dir=ios-onnxruntime/$onnxruntime_version
 
-SHERPA_ONNX_GITHUB=github.com
-
-if [ "$SHERPA_ONNX_GITHUB_MIRROW" == true ]; then
-    SHERPA_ONNX_GITHUB=hub.nuaa.cf
-fi
-
 if [ ! -z CMAKE_VERBOSE_MAKEFILE ]; then
```

Shellcheck (0.11.0)

[error] Argument to -z is always false due to literal strings.

(SC2157)